src/services/Audius/identity/index.ts

```diff
@@ -0,0 +1,41 @@
+import { AudiusClient } from '../AudiusClient'
+
+import BN from 'bn.js'
+import { Address } from 'types'
+
+export default class Identity {
+  public aud: AudiusClient
+
+  constructor(aud: AudiusClient) {
+    this.aud = aud
+  }
+
+  // Retrieves the minimum delegation set in identity for a service provider
+  async getMinimumDelegationAmount(
+    serviceProviderWallet: Address
+  ): Promise<BN | null> {
+    try {
+      const {
+        minimumDelegationAmount
+      } = await this.aud.libs.identityService.getMinimumDelegationAmount(
+        serviceProviderWallet
+      )
+      return new BN(minimumDelegationAmount)
```

Comment:
`new Web3.utils.BN('asdf')`

doesn't throw. maybe we could do some quick validation here? 🤷